### DISCUSSION TITLE
Text overflow in Settings panel

### DISCUSSION THREAD

**Discussion by kj415j45** [ppy/osu#16013] [Q&A] [answered]
Version: 2021.1207.0-lazer

en:
![image](https://user-images.githubusercontent.com/18349191/145323063-5d5bfc7c-6e4a-41ad-be3e-04f036bc06fe.png)
string: `Using unlimited frame limiter can lead to stutters, bad performance and overheating. It will not improve perceived latency. "2x refresh rate" is recommended.`

---

zh:
![image](https://user-images.githubusercontent.com/18349191/145323001-f7dae94b-1952-4c76-a477-71c88d1ca102.png)
string: `启用无限制帧数选项可能导致游戏卡顿、性能下降和电脑过热。这并不会降低体感延迟。推荐使用“2倍刷新率”选项。`

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/14837
